azure-pipelines.yml

```
@@ -4,16 +4,95 @@
 # https://aka.ms/yaml
 
 trigger:
-- main
+- master
 
-pool:
-  vmImage: 'ubuntu-latest'
+variables:
+   terraformVersion: 0.14.4
+   timeoutInMinutes: 60
+   agentPool: 'ubuntu-18.04'
+   service_connection: 'DTS-CFTPTL-INTSVC'
+   storage_account_rg: 'core-infra-intsvc-rg'
+   storage_account_name: 'cftptlintsvc'
+   component: 'terraform-github-management'
```

shall we not call this github?

what about?
```suggestion
   component: 'offboarding'
```